pkgs/development/python-modules/aiohttp/default.nix

```diff
@@ -46,6 +48,11 @@ buildPythonPackage rec {
       --replace "charset-normalizer >=2.0, < 3.0" "charset-normalizer >=2.0, < 4.0"
   '';
 
+  nativeBuildInputs = [
+    setuptools
+    wheel
```

Comment:
@FRidh Why `wheel`? It's not specified in upstream's `pyproject.toml`.

Reply:
The wheel package is always needed to make a wheel. Maybe we get away with it because pip that we instruct to build the wheel brings it in. I don't know anymore.